setup.py

```diff
@@ -3,7 +3,7 @@
 #===================================================================================================
 # Imports
 #===================================================================================================
-from distutils.core import setup
+from setuptools import setup
```

Comment:
Good point. I was thinking it's only a dev dependency but someone might for some inexplicable reason run setup.py manually. We're only using setuptools to be able to build the wheels